CHANGELOG.md

```diff
@@ -35,9 +47,21 @@ Branch: `nilrt/master/next`
 - [Deprecated](https://github.com/ni/meta-nilrt/pull/663) the `ntpdate` package, since it has been dropped by upstream.
 - [Removed](https://github.com/ni/meta-nilrt/pull/673/commits/01ac041e848b67553001cee00336637eefb9384a) NIcurl-specific functions from curl, since these sources are no longer used by NIcurl.
 - [Removed](https://github.com/ni/meta-nilrt/pull/675) the unused `efifix` scripts.
+- [Removed](https://github.com/ni/meta-nilrt/pull/744) quagga entry from files group.
+- [Removed](https://github.com/ni/meta-nilrt/pull/749) sqlite3.
+- [Removed](https://github.com/ni/meta-nilrt/pull/754) unused sudo access for lvuser.
+- [Removed](https://github.com/ni/meta-nilrt/pull/766) the ni-wireless-cert* IPKs from images.
+- [Disabled](https://github.com/ni/meta-nilrt/pull/767) DOCKER_BUILDKIT builds of images.
+- [Removed](https://github.com/ni/meta-nilrt/pull/769) deprecated skyline clients from the base system image.
+- [Removed](https://github.com/ni/meta-nilrt/pull/783) static linking of expat from libfmi.
 
 #### Fixed
 - [Fixed](https://github.com/ni/meta-nilrt/pull/676) a parsing error in the recovery media provisioning scripts.
+- [Fixed](https://github.com/ni/meta-nilrt/pull/750) the grep warnings getting thrown on boot.
+- [Fixed]() sysconfig settings to prevent users from accidentally enabling the UI in the SNAC configuration.
```

Comment:
link?

Reply:
No need to mention SNAC related changes I think unless it affects regular BSI as well.